rules/integrations/azure/credential_access_entra_signin_brute_force_microsoft_365.toml

```diff
@@ -2,8 +2,11 @@
 creation_date = "2024/09/06"
 integration = ["azure"]
 maturity = "production"
+min_stack_comments = "ES|QL not available until 8.13.0 in technical preview."
+min_stack_version = "8.13.0"
```

Comment:
I think we no longer release to 8.13 so we may be able to remove this min_stack. But thanks for adding it! :rocket: